components/definition/plots/NetPyNEPlots.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import IconMenu from 'material-ui/IconMenu';
+import Card, { CardHeader, CardText } from 'material-ui/Card';
+import Utils from '../../../Utils';
+import NetPyNEPlot from './NetPyNEPlot';
+import NetPyNENewPlot from './NetPyNENewPlot';
+import NetPyNEPlotThumbnail from './NetPyNEPlotThumbnail';
+
+export default class NetPyNEPlots extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedPlot: undefined,
+    };
+    this.selectPlot = this.selectPlot.bind(this);
+    this.handleNewPlot = this.handleNewPlot.bind(this);
+  };
+
+  selectPlot(plot) {
+    this.setState({ selectedPlot: plot });
+  };
+
+  handleNewPlot(plot) {
+    if (this.state.value!=undefined) {
+        var model = this.state.value;
+        model[plot] = true;
+    } else {
+        var model = {plot : true}
+    };
+    Utils
+      .sendPythonMessage("netpyne_geppetto.getAvailablePlots", [])
+      .then((response) => {
+        if (response.includes(plot)) {
```

Comment:
this method just ask to the backend if the current plot has already been defined. If it is not, then it will be created. If it was, then no further action is required.